src/Simulation/Core/IAzureSubmitter.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation.
```

Comment:
Now that this [PR](https://github.com/microsoft/qsharp-runtime/pull/629) has been merged, should _IAzureSubmitter.cs_ and _IQirSubmitter.cs_ be placed in the _Submitter_ folder?